doc/whatsnew/2.0.rst

```diff
@@ -25,6 +25,24 @@ Summary -- Release highlights
 
 New checkers
 ============
+* A new check was added, ``comparison-with-callable``.
+
+  This refactoring message is emitted when a comparison involves callable without parenthesis, which if compared
```

Comment:
I'd rewrite this as:
`This refactoring message is emitted when pylint detects that a comparison with a callable was made, which might suggest that some paranthesis were omitted, resulting in potential unwanted behaviour`.